recipes/azure-mgmt/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.20.1" %}
+
+package:
+  name: azure-mgmt
+  version: {{ version }}
+
+source:
+  fn: azure-mgmt-{{ version }}.zip
+  url: https://pypi.io/packages/source/a/azure-mgmt/azure-mgmt-{{ version }}.zip
+  md5: 8a678938c5e3f809392fe23e3653e83f
+  
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - azure-mgmt-compute ==0.20.1
+    - azure-mgmt-network ==0.20.1
+    - azure-mgmt-resource ==0.20.1
+    - azure-mgmt-storage ==0.20.0
+  run:
+    - python
+    - azure-mgmt-compute ==0.20.1
+    - azure-mgmt-network ==0.20.1
+    - azure-mgmt-resource ==0.20.1
+    - azure-mgmt-storage ==0.20.0
```

Comment:
@jakirkham that's something I'll try to keep in mind. I reckon that indeed they have version numbers that somewhat increase in tandem, so this could save some work